lua/nvim-highlight-colors/init.lua

```diff
@@ -20,7 +20,9 @@ local options = {
 	enable_tailwind = false
 }
 
-function is_window_already_created(row, value)
+local H = {}
```

Comment:
I am currently using M for the rest of the plugin, could you please change it to M as well?

Reply:
I used `H` because it's not the 'module' table, it's a 'helpers' table. The table will not be returned, it's local to the script, I actually removed `M` from that script (it's only returning some functions).

Anyway I can change it to `M` if you prefer `M`.

Reply:
Ah okay, that makes 😀
This is my first plugin so i am not really familiar with lua and how neovim works 👍
Please ignore this change

Reply:
It's a very good plugin, thanks for making it! It might be the best color highlighter for neovim out there atm.

Reply:
Ok I use `M` now.